tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
You are messing with me, right? Isn't it better to only import what's necessary unless you import very many subs so that a package prefix on call makes distinguishing easier?